packaging/nnstreamer.spec

```diff
@@ -1,7 +1,24 @@
-# Execute gbs with --define "testcoverage 1" in case that you must get unittest coverage statistics
+###########################################################################
+#
+#              Options for gbs/rpmbuild users
+#
+# gbs --define "unit_test 1"
+#       Execute all unit test cases
+#
+# gbs --define "testcoverage 1"
```

Comment:
```suggestion
# gbs build --define "testcoverage 1"
```
In addition, how about using "test_coverage and unit_test" or "testcoverage and unittest" instead of "testcoverage and unit_test"? This comment might be not directly related to your PR but I would suggest:)

Reply:
Renaming such will require updates of TAOS-CI and scripts in our CI server. :(
Let's consider it later.